### PR TITLE
chore: Add troubleshooting section to EAP page

### PR DESCRIPTION
**Pull request by shadrina**

### CODE REVIEW

File: docs/topics/install-eap-plugin.md [hunk 1]
```diff
@@ -24,3 +24,10 @@ You can follow these instructions to install [the preview version of the Kotlin
 4. Click **Install**. 
 
 If you want to work on existing projects that were created before installing the EAP version, you need to [configure your build for EAP](configure-build-for-eap.md). 
+
+## If you run into any problems
+
+* Report issues to [our issue tracker, YouTrack](https://kotl.in/issue).
```
Comment (koshachy) [inline docs/topics/install-eap-plugin.md:30]:
```suggestion
* Report an issue to [our issue tracker, YouTrack](https://kotl.in/issue).
```

File: docs/topics/install-eap-plugin.md [hunk 2]
```diff
@@ -24,3 +24,10 @@ You can follow these instructions to install [the preview version of the Kotlin
...
+* Find help in the _#eap_ channel on [Slack](https://kotlinlang.slack.com/) ([get an invite](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up)).
```
Comment (koshachy) [inline docs/topics/install-eap-plugin.md:31]: It's better to format this text in the same style, like in `Getting Started with Kotlin` (getting-started.md):
* #eap as a link (and not an italic)
* Slack as a plain text (not a link)
The link to the channel is more important than to the Slack

File: docs/topics/install-eap-plugin.md [hunk 3]
```diff
@@ -24,3 +24,10 @@ You can follow these instructions to install [the preview version of the Kotlin
...
+* Roll back to the latest stable version by selecting the **Stable** update channel in
```
Comment (koshachy) [inline docs/topics/install-eap-plugin.md:32]: This is a good one, but we also can get rid of:
 * **Gerund**: Roll back to the latest stable version – In **Tools | Kotlin | Configure Kotlin Plugin Updates** select the **Stable** update channel and click **Install**.
* **Repetition**: **Install** to install, roll back to the lastest stable ... to install the latest version.

**Review by koshachy — APPROVED** [review]
Approved with some minor comments.
Please take a look

File: docs/topics/install-eap-plugin.md (hunk 2 above)
Comment (shadrina) [reply docs/topics/install-eap-plugin.md:31]: I synchronized the formatting with other EAP-related pages. Do I need to update the formatting there too?

Comment (koshachy) [reply docs/topics/install-eap-plugin.md:31]: Yeah,
You can make this whole sentence as a link: [#eap channel in Kotlin Slack]

File: docs/topics/install-eap-plugin.md (hunk 3 above)
Comment (shadrina) [reply docs/topics/install-eap-plugin.md]: Thanks! Updated. I'll proofread the new sentence and then merge.

**Pull request merged**